docs/architecture.md

```diff
@@ -32,3 +32,11 @@ related:
 * {{ "lobby" | dfn }}
 * {{ "reception" | dfn }}
 * {{ "room" | dfn }}
+
+## Stages
+
+* rough-in or cut-out
+* fit-off or fit-out
```

Comment:
There appears to be a definition for this term without the hyphen, so wondering if should link to that and perhaps show both terms without a hyphen?

```suggestion
* {{ "fit out" | dfn }} or fit off
```